csb/src/main/scala/edu/msstate/dasi/csb/Neo4jWorkload.scala

```diff
@@ -275,8 +275,9 @@ class Neo4jWorkload(url: String, username: String, password: String) extends Wor
    * Computes the shortest path from a source vertex to all other vertices.
    */
   def sssp[VD: ClassTag, ED: ClassTag](graph: Graph[VD, ED], src: VertexId): Unit = {
-    val query ="MATCH (src {name:\"" + src + "\"}), (dst)," +
-      "path = shortestPath((src)-[*]->(dst))" +
+    val query ="MATCH (src {name:\"" + src + "\"}), (dst) , (path)" +
+      "WHERE dst.name <> src.name AND " +
+      "path=shortestPath((src)-[*]->(dst)) " +
       "RETURN src, dst, path;"
```

Comment:
I'm not sure the new query is correct, the ``(path)`` on the first line will be treated as a node pattern, so you are actually saying to match three nodes at the same time.

I also tried to execute it on a small graph and it doesn't return the result that I would expect.

Try instead:
````
val query ="MATCH (src {name:\"" + src + "\"}), (dst), " +
  "path = shortestPath((src)-[*]->(dst)) "
  "WHERE dst.name <> src.name "
  "RETURN src, dst, path;"
````